### PR TITLE
Azure CI: Add timeouts, *smoke* filter

### PR DESCRIPTION
**Pull request by azhogov**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -21,6 +22,8 @@ jobs:
   - script: |
       sudo apt --assume-yes install libusb-1.0-0-dev
       python3 -m pip install -r ./inference-engine/ie_bridges/python/requirements.txt
+      # For Python API tests, should be in requirements.txt
+      python3 -m pip install pytest==5.4.2
```
Comment (SDxKeeper) [inline azure-pipelines.yml:26]: there are should be dedicated requirements.txt file in tests folder

File: azure-pipelines.yml [hunk 2]
```diff
@@ -90,9 +94,20 @@ jobs:
     workingDirectory: dldt-build
     displayName: 'IE CAPITests'
     continueOnError: false
+#  - script: |
```
Comment (SDxKeeper) [inline azure-pipelines.yml:97]: is it draft pull request?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -170,7 +186,7 @@ jobs:
   - script: |
       export DATA_PATH=`pwd`/../testdata
       export MODELS_PATH=`pwd`/../testdata
-      ../bin/intel64/$(BUILD_TYPE)/MklDnnFunctionalTests --gtest_filter=-smoke_MobileNet/ModelTransformationsTest.LPT/mobilenet_v2_tf_depthwise_batch1_inPluginDisabled_inTestDisabled_asymmetric*
+      ../bin/intel64/$(BUILD_TYPE)/MklDnnFunctionalTests --gtest_filter=*smoke*:-smoke_MobileNet/ModelTransformationsTest.LPT/mobilenet_v2_tf_depthwise_batch1_inPluginDisabled_inTestDisabled_asymmetric*
```
Comment (SDxKeeper) [inline azure-pipelines.yml:189]: can we get rid of relative paths one folder up?

Comment (azhogov) [reply azure-pipelines.yml:189]: yes, later

File: azure-pipelines.yml (hunk 2 above)
Comment (azhogov) [reply azure-pipelines.yml:97]: partially, but good enough for next step

File: azure-pipelines.yml (hunk 1 above)
Comment (azhogov) [reply azure-pipelines.yml:26]: agree, need a request on developers

Comment (akuporos) [reply azure-pipelines.yml:26]: We have special requirements-dev.txt: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/ie_bridges/python/src/requirements-dev.txt

**Pull request merged**